### DISCUSSION TITLE
Web App Laravel - Cannot read properties of undefined

### DISCUSSION THREAD

**Discussion by jrendonzam** [ionic-team/capacitor#7593] [Q&A]
Hello, I am trying to use Capacitor in my web app developed in Laravel, when I run the project within Android Studio it does not open the application, what opens is Chrome with the URL of my web app. When I see the console it gives me this error Capacitor/Console - E File: - Line 1 - Msg: Uncaught TypeError: Cannot read properties of undefined (reading 'triggerEvent'). Will you know what it is due to?
![error](https://github.com/user-attachments/assets/71774851-2bf0-42d3-b649-ccc51ea953aa)

**Comment by Ironmunkii** [comment]
I have the same issue. @jrendonzam did you find a solution?
